crates/biome_service/src/file_handlers/javascript.rs

```diff
@@ -659,12 +659,24 @@ pub(crate) fn fix_all(params: FixAllParams) -> Result<FixFileResult, WorkspaceEr
                 }
 
                 for action in signal.actions() {
-                    // suppression actions should not be part of the fixes (safe or suggested)
-                    if action.is_suppression() {
-                        continue;
+                    if action.is_suppression()
+                        && params.suppress
+                        && action.applicability == Applicability::Always
+                        // TODO: I was getting suppressions written for nursery rules, but I don't know
+                        // why so I hacked this.
+                        && action
+                            .rule_name
+                            .is_some_and(|(first, _)| first != "nursery")
```

Comment:
Safe to say this isn't right, but it stopped a bunch of ignores getting written for `lint/nursery/useStrictMode`. I don't know if it's a bug with that rule because there aren't any other nursery rules getting written from what I can tell.

Reply:
Yeah this is definitely weird to me. I'm assuming `useStrictMode` is disabled in your config, right? I would've thought that disabled rules wouldn't get evaluated (and therefore not emit diagnostics at all).

Reply:
Correct, no nursery rules in our configuration.

Reply:
Nursery rules are enabled in debug builds, but they are turned off in production builds. So if a nursery rule is *recommended* and it has a fix, it's normal to see the changes.

Reply:
Oh, TIL! That explains that then. Removing this.